### PR TITLE
[BugFix] Ensure image overlay announces as image details

### PR DESCRIPTION
**Pull request by emlynmac**
# What
Opening the image overlay, voiceover / narration did not announce the presence of the overlay.

Updates to announcement have been added, along with a localization string to provide a means to configure the announcement message.

# Why
Increase usability of the image overlay component for users who rely on screen readers or other assistive technologies.

# How Tested
Manual testing on Windows 11 with narrator and interop. scenario

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: change-beta/@azure-communication-react-31153022-b68a-433b-87c3-d55c675d6e37.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "area": "fix",
+  "workstream": "A11y",
+  "comment": "Update imageoverlay to announce when visible",
```
Comment (vhuseinova-msft) [inline change-beta/@azure-communication-react-31153022-b68a-433b-87c3-d55c675d6e37.json:5]:
```suggestion
  "comment": "Update image overlay to announce when visible",
```

File: change/@azure-communication-react-31153022-b68a-433b-87c3-d55c675d6e37.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "area": "fix",
+  "workstream": "A11y",
+  "comment": "Update imageoverlay to announce when visible",
```
Comment (vhuseinova-msft) [inline change/@azure-communication-react-31153022-b68a-433b-87c3-d55c675d6e37.json:5]:
```suggestion
  "comment": "Update image overlay to announce when visible",
```

File: packages/react-components/src/components/ImageOverlay.tsx [hunk 3]
```diff
@@ -106,7 +111,7 @@ export const ImageOverlay = (props: ImageOverlayProps): JSX.Element => {
 
   const renderHeaderBar = (): JSX.Element => {
     return (
-      <Stack className={mergeStyles(headerStyle)} role="heading" aria-label={title || 'Image'} aria-level={2}>
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/ImageOverlay.tsx:109]: Do we still need it? 
It looks like it wasused during the keyboard navigation

File: packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx [hunk 4]
```diff
@@ -744,6 +744,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       {overlayImageItem && (
         <ImageOverlay
           {...overlayImageItem}
+          altText={'Image Details Opened'}
```
Comment (vhuseinova-msft) [inline packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx:747]: It probably needs to be localizable

Comment (emlynmac) [reply packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx:747]: I can remove this actually, thanks for catching.

Comment (dmceachernmsft) [reply packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx]: You could maybe also just re-use the aria-label here too

File: packages/react-components/src/components/ImageOverlay.tsx (hunk 3 above)
Comment (dmceachernmsft) [reply packages/react-components/src/components/ImageOverlay.tsx]: you might need the role and the level but if the announcer is using the string then the label can be removed here.

**Comment by emlynmac** [comment]
Adding breaking changes label. There is an additional localization string that is defaulted as part of the release process.

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/react-components/src/components/ImageOverlay.tsx [hunk 5]
```diff
@@ -178,6 +183,7 @@ export const ImageOverlay = (props: ImageOverlayProps): JSX.Element => {
       styles={{ main: focusTrapZoneStyle, scrollableContent: scrollableContentStyle }}
       isDarkOverlay={true}
     >
+      <Announcer polite={{ id: `${title}`, message: localeStrings.overlayVisibleMessage ?? altText }} />
```
Comment (JoshuaLai) [inline packages/react-components/src/components/ImageOverlay.tsx:186]: Do we want to introduce a new string for this? Just a call out if we are going have `altText` read out as a default we may want to document it so developers are clear about this behaviour

**Review by carocao-msft — APPROVED** [review]

**Pull request merged**